packages/common/src/hardhat/tasks/artifacts.ts

```diff
@@ -188,7 +188,7 @@ export type { ${normalizeClassName(contractName)}Web3Events };\n`
       fs.appendFileSync(out, `  ${contractName}: "${relativePath}",\n`)
     );
     fs.appendFileSync(out, "};\n");
-    fs.appendFileSync(out, "type ContractName = keyof typeof artifactPaths;\n");
+    fs.appendFileSync(out, "export type ContractName = keyof typeof artifactPaths;\n");
```

Comment:
I thought I needed this type in `InsuredBridgeL1Client` but I realized that instead of making the user specify the type of `BridgeAdmin` (OVM vs. AVM), we could just use the `BridgeAdminInterface` if we made a small tweak to the interface. More on this contract change below.